package.json

```diff
@@ -19,7 +19,7 @@
     "build:happ": "sh ./scripts/build-bundle.sh",
     "run:conductor": "hc sandbox clean && hc sandbox generate ./tests/integration/setup/bundle/elemental-chat.happ -r=8888 -a='elemental-chat:alpha19:0001'",
     "build:holo-host": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://chaperone.holo.host/' vue-cli-service build",
-    "build:holo-dev": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://qa-chaperone-rsm.holotest.net/' vue-cli-service build --mode development",
+    "build:holo-dev": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://devnet-chaperone.holo.host/' vue-cli-service build --mode development",
```

Comment:
This needs to be `'https://chaperone-rsm.devnet.holotest.net/`

Reply:
@robbiecarlton @SirRobLyon

Reply:
Just for reference, these are the chaperone URLs as implemented in publisher-portal: https://github.com/Holo-Host/publisher-portal/blob/develop/package.json#L9-L11